addons/playersuppression/functions/fnc_modulePlayerSuppression.sqf

```diff
@@ -0,0 +1,38 @@
+#include "script_component.hpp"
+
+private ["_logic"];
+_logic = _this select 0;
+GVAR(intensity)         = _logic getVariable "intensity";
+GVAR(enablePlayerSuppression) = true;
```

Comment:
You probably want to use the ace_settings system for this, so it can be set by module or server config.

e.g.:
```
class ACE_Settings {
    class GVAR(EnableCombatDeafness) {
        value = 1;
        typeName = "BOOL";
    };
```